test/passing/js_source.ml.ref

```diff
@@ -3189,12 +3189,10 @@ let sort (type s) (module Set : Set.S with type elt = s) l =
 (* No real improvement here? *)
 let make_set (type s) cmp =
   (module Set.Make (struct
-    type t = s
+     type t = s
```

Comment:
Note that in general the output produced for cases where a paren precedes a keyword indents from the paren, not the keyword, so this added indentation is in conflict with that. The choice of where to indent from can be discussed, but I don't think it shouldn't be changed just in this case.

Reply:
The `end` is also indented to be aligned with the `module` keyword. Should I undo this and open a new PR ?

Reply:
I removed that indentation.